src/app/components/panels/panel-switcher.component.scss

```diff
@@ -4,7 +4,7 @@
 .left-panel-bar {
   display: flex;
   flex-direction: column;
-  height: calc(100% - #{$main-header-height});
+  height: calc(100% - #{$main-header-height} - 26px);
```

Comment:
Does the 26px represent the `.toggle-panel-left` height here? It could be put into a variable so if it changes in the future, you only have to change it in one place.